registrations/tasks.py

```diff
@@ -399,14 +399,23 @@ class BasePushRegistrationToJembi(object):
     name = "ndoh_hub.registrations.tasks.base_push_registration_to_jembi"
     l = get_task_logger(__name__)
 
-    def get_patient_id(self, id_type, id_no=None,
-                       passport_origin=None, mom_msisdn=None):
+    def get_patient_id(self, id_type, id_no=None, passport_origin=None,
+                       mom_msisdn=None, registrant_id=None):
         if id_type == 'sa_id':
             return id_no + "^^^ZAF^NI"
         elif id_type == 'passport':
             return id_no + '^^^' + passport_origin.upper() + '^PPN'
-        else:
+        elif mom_msisdn:
             return mom_msisdn.replace('+', '') + '^^^ZAF^TEL'
+        elif registrant_id:
+            identity = is_client.get_identity(registrant_id)
+            if identity:
```

Comment:
Yes, it will then fail on Jembi's side with a error we can use, previously it failed because mom_msisdn was None, the errors in the log files weren't helpful and it never got to Jembi. Is this ok?